extension.neon

```diff
@@ -26,13 +26,7 @@ parameters:
         - Illuminate\Http\Request
     excludes_analyse:
         - *.blade.php
-    ignoreErrors:
-        - '#Result of function abort \(void\) is used#'
```

Comment:
Is this a common thing people do? It looks like the result of `abort()` actually **is** `void`.

Reply:
Yes. I've seen people using `return abort(401)` before. See [here](https://grep.app/search?q=return%20abort%28&filter[lang][0]=PHP) for examples.

Reply:
It kind of makes the code more clear, i can see why someone would like to do it. Still, i would let that up to personal preference and not mention it in the docs, as it is technically wrong.

Reply:
Let's tell `return abort`-ers to stop doing it.